default.nix

```diff
@@ -45,12 +45,14 @@ in rec {
 
   mkYarnModules = {
     name,
+    pname,
```

Comment:
in that case let's remove the name argument and also pass the version along the `pname`